CONTRIBUTING.md

```diff
@@ -86,7 +88,50 @@ cd ../<yourproject>/
 yarn start
 ```
 
-You can now hack in the cloned repository of `api-platform-admin`.
+> You can now hack in the cloned repository of `api-platform-admin`.
+
+#### Running Admin through Storybook
+
+If you don't have an existing API Platform application, you can use one of the bundled example APIs, and visualize the admin through [Storybook](https://storybook.js.org/).
+
+Get the source of `@api-platform/admin`:
+
+```shell
+cd ..
```

Comment:
`cd ..` doesn't seem necessary to me since we won't need to move back and forth between two projects in this case, we only work with the `admin` repo